refreshdata.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+""" Refresh data files for the Covid19 Mtl dashboard """
+
+import os
+import sys
+
+from datetime import datetime
+from argparse import ArgumentParser
+
+import requests # TODO: add these to requirements.in after merging with merwork
+import lxml
+import bs4
+
+# Data sources mapping
+# {filename: url}
+SOURCES = { #'cases.csv': '', 
+            #'cases_per1000': '', 
+            #'data_mtl': '',
+
+            # The main page for this dataset is https://www.inspq.qc.ca/covid-19/donnees 
+            'data_qc.csv': 'https://www.inspq.qc.ca/sites/default/files/covid/donnees/combine.csv'}
+DATA_DIR = '/tmp/data'
```

Comment:
Suggest to use the `tempfile` module

Reply:
> Could you add a few lines to README about intended usage?
> Is it supposed to be run once per day then files added to repo?
> We talked about a scheduled job on heroku, but the filesystem is volatile, so will the files need to be saved to S3 or Digital Ocean storage?

I'm in favour of keeping this as a command line tool/script, at least until it is feature complete (and yes, adding the files to the repo/deploying once a day). I will still need to manually enter the other data sources not covered by this data scrapper until we can cover all the data sources, so a hybrid model where I can save time with the scrapper but still fill in the blanks manually would be beneficial. We are also adding new data sources with #15 and #19, so I think it is good to keep this flexible and modular at this point. We can modify to automatically upload data to some form of object storage (S3/GCP) later on.